### DISCUSSION TITLE
Enabling Sponsorship in the most efficient manner

### DISCUSSION THREAD

**Discussion by SQLDBAWithABeard** [dataplat/DataSaturdays#81] [Q&A]
I have heard from some people who are involved in the marketing teams for vendors that we know and love and thank for sponsoring our events. They have mentioned that a single point of entry for sponsorship is an important consideration.

It would be useful to have some of the vendors, in addition to event organisers, provide some suggestions that this community-based event creation solution can take forward to best provide the most benefit and easiest to manage solutions for both the **Sponsors** *and the* **Event organisers**

If they are not willing to comment publicly at this time, feel free to contact myself or another of the admins and we will add their suggestions to this discussion.

It would be best if this discussion does not get muddied with the #55 discussion about the legal side of things but focuses more on the required, lets call it tooling and process for want of better terminology.

**Comment by MarkDataGuy** [comment]
Hi. It may be difficult to seperate aspects of this from legal as if you plan to collect money, then you need to have some legal basis in order to have a stripe or PayPal  account, and then you need to transfer that money on. If datasaturday is collecting money then I imagine you end up in tax/vat/accounting territory. Instead you need to allow a user group use the site to raise an invoice and collect money via their own entity.

Any sponsors I have dealt with need an invoice and a method of paying, either via credit card or EFT. In my case, I have a club bank account in existence for my user group so I can raise my own invoice ( via accounting software ) and collect payment via CC using the same software. However, where this is not possible, the ability to raise an invoice via datasaturday would be useful. If the same invoice can be configured to show the user groups bank details or url link to UG's paypal or stripe account then you would be able to allow groups to use datasaturday to invoice and  collect money. 

This mechanism used to exist on sqlsaturday, it was a bit cumbersome but it did produce a basic invoice and url to the UG PayPal link. 

I hope that helps.

Mark

**Comment by way0utwest** [comment]
In terms of tooling, I would say that we want to have a linkage from an event to something like a Paypal/Stripe/etc, that can receive money. Some events/organizers have a non profit, some use personal accounts, but there needs to be a way to allow customization here.

For the toolset, I would say that we need to support vendors:
- Stripe
- Paypal
- Venmo
- ???

In terms of the tooling, we also need a way to produce an invoice or payment request given some data (assuming this isn't stored) likely means
- some form gathering data
- some processing of the form into a formal invoice document
- database storage?

Perhaps an API can evolve, but I would make this simple early on, perhaps just a simple HTML processing of data into a nice looking form that could be rendered. Perhaps even a markdown PR that produces an invoice for a particular event/sponsor?
